airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/CheckAndRemoveRecordWriter.java

```diff
@@ -14,6 +14,6 @@ public interface CheckAndRemoveRecordWriter {
    * different, then the staging writer corresponding to `stagingFileName` is closed and the name of
    * the new file where the record will be sent will be returned.
    */
-  String apply(AirbyteStreamNameNamespacePair airbyteStreamNameNamespacePair, String stagingFileName) throws Exception;
+  String apply(AirbyteStreamNameNamespacePair stream, String stagingFileName) throws Exception;
```

Comment:
this is kind of confusing because technically this is a tuple of <stream, fully qualified namespace> so somewhat in favor of not having this be the new name

Reply:
We can come up with a better name as well. streamName? streamIdentifier?

Reply:
fwiw I've used streamIdentifier in the past, since each AirbyteStreamNameNamespacePair uniquely identifies a stream (and it's not overloaded with the stream itself having a name)